docs/source/common/early_stopping.rst

```diff
@@ -62,8 +67,11 @@ Additional parameters that stop training at extreme points:
 - ``stopping_threshold``: Stops training immediately once the monitored quantity reaches this threshold.
   It is useful when we know that going beyond a certain optimal value does not further benefit us.
 - ``divergence_threshold``: Stops training as soon as the monitored quantity becomes worse than this threshold.
-  When reaching a value this bad, we believe the model cannot recover anymore and it is better to stop early and run with different initial conditions.
-- ``check_finite``: When turned on, we stop training if the monitored metric becomes NaN or infinite.
+  When reaching a value this bad, it believes the model cannot recover anymore and it is better to stop early and run with different initial conditions.
```

Comment:
no, the "we" in the language here was the correct expression! "we" is the user who is putting these values into the callback.